### DISCUSSION TITLE
Want to see total gas spend on zksync era like eth chain

### DISCUSSION THREAD

**Discussion by getairdrop24** [zkSync-Community-Hub/zksync-developers#357] [Polls]
![Screenshot 2024-02-20 193154](https://github.com/zkSync-Community-Hub/zksync-developers/assets/85836509/199e4d55-1c0b-41cc-90a2-5a7c19928a98)
By the help of this feature, users can easily track their spent gas on the ZKSync chain without the need for third-party resources.

**Comment by wunrorobudaniel** [comment]
Enhance Visibility: With the new gas tracking feature, users gain transparency into their gas expenditures on the ZKSync chain, empowering them to monitor and manage their transactions efficiently.

**Reply by getairdrop24** [reply]
where can i found this feature.

**Comment by sarahzitzner** [comment]
Gas tracking feature added for enhanced user experience and transparency on ZKSync chain.

**Comment by deanhobson** [comment]
The feature being described allows users to track their gas usage on the ZKSync chain directly, without the need for external tools or resources. This provides a more streamlined and convenient experience for users, as they can easily monitor their gas usage without leaving the ZKSync ecosystem.
